test/run.js

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env node
+
+const { spawnSync } = require('child_process')
+const { readdirSync } = require('fs')
+const { extname, join } = require('path')
+
+process.env.NODE_ENV = 'test';
+
+process.stdout.write('\n')
+readdirSync(__dirname).forEach((file) => {
+  if (!file.startsWith('test.') || extname(file) !== '.js')
+    return;
+  process.stdout.write(`\x1b[90m   ${file}\x1b[0m `);

```

Comment:
Does Windows support ANSI colors in terminal?

Reply:
Yes

Reply:
Since Win10 terminal control sequences are supported natively by the console. For previous Windows versions Node (libuv to be more precise) has special emulation layer that translates the codes to winapi calls.